src/main/java/gregtech/loaders/recipe/MetaTileEntityMachineRecipeLoader.java

```diff
@@ -562,38 +563,56 @@ public static void init() {
                 .input(HULL[EV])
                 .input(frameGt, Titanium, 4)
                 .input(circuit, Tier.EV, 4)
-                .input(ELECTRIC_MOTOR_EV, 4)
+                .input(ELECTRIC_PISTON_EV, 4)
                 .input(ELECTRIC_PUMP_EV, 4)
                 .input(CONVEYOR_MODULE_EV, 4)
+                .input(SENSOR_EV, 1)
                 .input(gear, Tungsten, 4)
                 .circuitMeta(2)
                 .output(BASIC_LARGE_MINER)
                 .duration(400).EUt(VA[EV]).buildAndRegister();
 
         ASSEMBLER_RECIPES.recipeBuilder()
                 .input(HULL[IV])
-                .input(frameGt, TungstenSteel, 4)
+                .input(frameGt, HSSG, 4)
                 .input(circuit, Tier.IV, 4)
-                .input(ELECTRIC_MOTOR_IV, 4)
+                .input(ELECTRIC_PISTON_IV, 4)
                 .input(ELECTRIC_PUMP_IV, 4)
                 .input(CONVEYOR_MODULE_IV, 4)
+                .input(SENSOR_IV, 1)
                 .input(gear, Iridium, 4)
                 .circuitMeta(2)
                 .output(LARGE_MINER)
                 .duration(400).EUt(VA[IV]).buildAndRegister();
 
-        ASSEMBLER_RECIPES.recipeBuilder()
+        ASSEMBLY_LINE_RECIPES.recipeBuilder()
                 .input(HULL[LuV])
                 .input(frameGt, HSSS, 4)
                 .input(circuit, Tier.LuV, 4)
-                .input(ELECTRIC_MOTOR_LuV, 4)
+                .input(ELECTRIC_PISTON_LUV, 4)
                 .input(ELECTRIC_PUMP_LuV, 4)
                 .input(CONVEYOR_MODULE_LuV, 4)
+                .input(SENSOR_LuV, 1)
                 .input(gear, Ruridit, 4)
-                .circuitMeta(2)
+                .fluidInputs(Lubricant.getFluid(1000), SolderingAlloy.getFluid(1152))
```

Comment:
this change might cause recipe conflict

Reply:
i figured it was fine since it is now an assline recipe. i could change it back to assembler with a circuit.